bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/handler/VenstarThermostatHandler.java

```diff
@@ -93,10 +108,10 @@
 
     private static final int TIMEOUT_SECONDS = 30;
     private static final int UPDATE_AFTER_COMMAND_SECONDS = 2;
-
```

Comment:
This was not bad to separate constants with a blank line.

Reply:
Not sure I understand sorry.

Reply:
This is minor, you suppressed a blank line that made a better separation between the constants and the class members.
This was just better before your change.

Reply:
ok